WordPress/Classes/ViewRelated/EEUUSCompliance/CompliancePopoverCoordinator.swift

```diff
@@ -21,20 +22,31 @@ final class CompliancePopoverCoordinator: CompliancePopoverCoordinatorProtocol {
 
     // MARK: - Init
 
-    init(defaults: UserDefaults = UserDefaults.standard) {
+    init(
+        defaults: UserDefaults = UserDefaults.standard,
+        complianceService: ComplianceLocationService = .init(),
+        isFeatureFlagEnabled: @autoclosure @escaping () -> Bool = FeatureFlag.compliancePopover.enabled
```

Comment:
Why is a closure needed for this? Why not pass the value directly?
You can also pass an instance of `RemoteFeatureFlagStore` similar to `JetpackBrandingMenuCardPresenter`.

Reply:
> Why not pass the value directly?

I previously did this, but an app restart was needed when the feature flag is toggled, that's because the feature flag value stored in `CompliancePopoverCoodinator` wont be updated when the feature flag is toggled. The closure approach fixes this problem, it works like a "computed property": 

```swift
var isFeatureFlagEnabled: Bool {
    return FeatureFlag.compliancePopover.enabled
}
```

> You can also pass an instance of RemoteFeatureFlagStore similar to JetpackBrandingMenuCardPresenter.

The `CompliancePopoverCoordinator` doesn't need the whole `RemoteFeatureFlagStore` object. Also, I think the `RemoteFeatureFlagStore` doesn't work with the local `FeatureFlag`.

Reply:
> Also, I think the RemoteFeatureFlagStore doesn't work with the local FeatureFlag

Oh, my bad. But since this is a local flag, this means we are using a closure only to make debugging easier since local flags can only change from the debug menu, correct?

Reply:
> this means we are using a closure only to make debugging easier since local flags can only change from the debug menu, correct?

Correct 👍

Reply:
Having this just to streamline debugging seems like a code smell to me. Having to relaunch the app for changes to be applied seems like a reasonable compromise, given it's a low-friction feature. And it's a behavior that exists in other places in the app, like the new tab bar icons, for example.

Reply:
Resolved in 3e144368c665803694b8b48b1006556e1b652e10